src/components/OrgFile/components/DescriptionEditorModal/index.js

```diff
@@ -17,6 +17,9 @@ export default class DescriptionEditorModal extends PureComponent {
       descriptionValue: props.editRawValues
         ? this.calculateRawDescription(props.header)
         : props.header.get('rawDescription'),
+      editorDescriptionHeightValue: props.editorDescriptionHeightValue
+        ? props.editorDescriptionHeightValue
+        : "8",
```

Comment:
When creating a new persistable field, we add it to https://github.com/200ok-ch/organice/blob/a7f57ce8520778625ba634617be681e2de6c117d/src/util/settings_persister.js#L74

There, you can also set a default.

This functionality is a bit hidden. Adding this to some part of the dev documentation could be helpful.

Reply:
Hah, I just saw that you actually added the flag there! Good job, so it wasn't too hidden for you(;

Just the default is missing.

Nice! :100: